Raven.NET.Core/Extensions/RavenSubjectExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Raven.NET.Core.Extensions
+{
+    public static class RavenSubjectExtensions
+    {
+        
+        /// <summary>
+        /// Create string based on object reflection to store current value in memory cache
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <param name="propName"></param>
+        /// <returns></returns>
+        public static string CreateCacheValue(this object obj, string propName = null)
+        {
+            var sb = new StringBuilder();
+            if (obj.GetType().IsValueType || obj is string)
```

Comment:
I'm more in favor of having if statements with brackets. Makes code a bit easier to read

Reply:
I guess for me inline is habbit :D haha